src/west/app/project.py

```diff
@@ -867,6 +867,16 @@ def fetch_missing_imports(self, args):
         self.manifest = Manifest.from_file(topdir=self.topdir,
                                            importer=self.update_importer)
 
+    def update_submodules(self, project, submodules, call_location):
+        # Updates given list of project submodules by using
+        # 'git submodule update --init' command from the location passed
+        # through the call_location parameter value.
+        if submodules:
+            # Iterate through the submodules list and update each one of them
```

Comment:
nit: I think the code is self-explanatory and doesn't need such a comment.

Reply:
Yeah, I agree it is a bit redundant comment.